docs/cel2/whats-changed/l1-l2.md

```diff
@@ -15,37 +15,35 @@ More details on supported transaction types [here](https://specs.celo.org/tx_typ
 
 ## Native bridge to Ethereum
 
-An integral part of L2 design is that they have a native bridge to ethereum. Celo will become an ERC20 token on Ethereum and users will be able to use the native bridge to bridge between the Celo L2 and Ethereum. The Alfajores testnet bridge can be accessed [here](https://testnets.superbridge.app/celo-alfajores).
+An integral part of L2 design is that they have a native bridge to Ethereum. Celo will become an ERC20 token on Ethereum and users will be able to use the native bridge to move between the Celo L2 and Ethereum. The Alfajores testnet bridge can be accessed [here](https://testnets.superbridge.app/celo-alfajores).
 
 ## Consensus
 
-The BFT consensus protocol will be removed and replaced with a centralized sequencer. Although there will be no more need for validators the validator set, election mechanism and voting will remain, however validators will not need to run any infrastructure for the L2.
+The BFT consensus protocol will be removed and replaced with a centralized sequencer. Although validators will no longer be needed to secure consensus, the election / voting mechanism and validator set will remain for the time being.
 
-This is a temporary situation, after Mainnet launch we will be looking at re-introducing active roles for validators.  
+This is a temporary situation, and we will be working on re-introducing active roles for validators after Mainnet launch. For now, validators do not need to run any special L2 infrastructure.
```

Comment:
This is a bit misleading, validators are expected to be community rpc nodes

Reply:
Yea, I wasn't sure the right way to address that. I'll add a note mentioning being full nodes